docs/src/demos/hooks/use-fullscreen.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useFullscreen } from '@mantine/hooks';
+import { Button, Paper, Text, useMantineTheme } from '@mantine/core';
+
+const code = `
+import { useFullscreen } from '@mantine/hooks';
+import { Button, Paper, Text, useMantineTheme } from '@mantine/core';
+
+function Demo() {
+  const theme = useMantineTheme();
+  const { toggle, fullscreen } = useFullscreen();
+  const { ref, toggle: toggleWithRef, fullscreen: fullscreenWithRef } = useFullscreen();
+
+  return (
+    <Paper elementRef={ref} style={{ height: 100 }}>
+      <div
+        style={{
+          height: '100%',
+          width: '100%',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          flexDirection: 'column',
+        }}
```

Comment:
Usually it's not a good idea to include inessential styles to demo to code – this makes demo cluttered and makes harder to understand features that are currently demonstrated, please remove styles and props (e.g. `size="lg"`) that are not associated with hook behavior from demo  code

Reply:
For the full screen example to look cleaner i included these styles, as when the component goes into full screen this correctly centers the buttons in the middle of the screen.
But can do!